music21/midi/translate.py

```diff
@@ -535,7 +538,9 @@ def midiEventsToChord(eventList, ticksPerQuarter=None, inputM21=None):
         offEvents = eventList[(len(eventList) // 2):]
         # first is always delta time
         tOn = onEvents[0].time
-        tOff = offEvents[0].time
+        # use the off time of the last chord member
+        # -1 is the event, -2 is the delta time for the event
+        tOff = offEvents[-2].time
```

Comment:
Are we absolutely sure that this will always be the case?  Just want to be totally sure.  I think so given the offEvents definition.

Reply:
Should check that there are indeed at least 4 events in eventList.